docs/concepts/data-partitions.md

```diff
@@ -0,0 +1,95 @@
+
+## Feature Enablement
+The data partitions feature can be enabled by setting the configuration key `DicomServer:Features:EnableDataPartitions` to `true` through your local [appsettings.json](../../src/Microsoft.Health.Dicom.Web/appsettings.json) file or host-specific options.
+
+Once enabled, this feature adds a new data partition resource, creates an initial partition named `Microsoft.Default`, and modifies the API surface of the DICOM server. 
+
+> *The data partition feature **cannot be disabled** once partitioned data is present - a `DataPartitionsFeatureCannotBeDisabledException` will be thrown at startup.*
+
+## API Changes
+All following URIs assume an implicit DICOM service base URI. For example, the base URI of a DICOM server running locally would be `https://localhost:63838/`. Example requests for new and modified APIs can be found in the [data partition Postman collection](../resources/data-partition-feature.postman_collection.json).
+
+### List Partitions
+Lists all data partitions.
+
+```http
+GET /partitions
+```
+
+#### Request Header
+
+| Name         | Required  | Type   | Description                     |
+| ------------ | --------- | ------ | ------------------------------- |
+| Content-Type | False     | string | `application/json` is supported |
+
+#### Responses
+
+| Name              | Type                          | Description                           |
+| ----------------- | ----------------------------- | ------------------------------------- |
+| 200 (OK)          | [Partition](#partition)`[]`   | A list of partitions is returned      |
+| 204 (No Content)  |                               | No partitions exist                   |
+| 400 (Bad Request) |                               | Data partitions feature is disabled   |
+
+### STOW Changes
+Once partitions are enabled, STOW requests **must** include a data partition URI segment after the base URI, of the form `/partitions/{partitionName}`, where `partitionName` is:
+ - Up to 64 characters long
+ - Composed of any combination of alphanumeric characters, `.`, `-`, and `_`, to allow both DICOM UID and GUID formats, as well as human-readable identifiers
+
+If a partition with the given valid name does not exist, it will be created **implicitly** by the STOW request. If the partition segment is not included, a 400 invalid request will be returned with a message indicating that a partition must be specified. 
+
+**Example:** A DICOM file is stored in the partition `myPartition-1`. If it doesn't already exist, it will be created.
```

Comment:
Maybe instead of "If it doesn't already exist..." -> "If the partition doesn't already exist..."

Reply:
Good clarification - updated.